vllm/v1/executor/abstract.py

```diff
@@ -95,12 +92,12 @@ class UniProcExecutor(UniProcExecutorV0, Executor):
 
 class ExecutorWithExternalLauncher(ExecutorWithExternalLauncherV0, Executor):
 
-    def determine_available_memory(self) -> int:  # in bytes
+    def determine_available_memory(self) -> List[int]:  # in bytes
```

Comment:
Why do we need to change the return type of this function?

Reply:
https://github.com/vllm-project/vllm/pull/14079/files#r1986211589